packages/marko-newsletters/express/template-router.js

```diff
@@ -23,6 +23,40 @@ query WithMarkoNewsletter($input: EmailNewsletterAliasQueryInput!) {
     site {
       ...MarkoWebsiteContextFragment
     }
+    alphaThemeConfigs {
```

Comment:
@B77Mills actually, it would probably be better if this was updated so a custom fragment can be _passed_ into the template loader. That way, you can request any fields you want, on the fly, when you need them (instead of statically setting them here). 

This is how the content middleware works for content pages in the website framework.